src/meta/meta_http_service.cpp

```diff
@@ -827,12 +828,26 @@ void meta_http_service::update_scenario_handler(const http_request &req, http_re
 
 bool meta_http_service::redirect_if_not_primary(const http_request &req, http_response &resp)
 {
-#ifdef DSN_MOCK_TEST
-    return true;
+#ifdef MOCK_TEST
+    // Once MOCK_TEST is defined, the meta server must has been built with `./run.sh build --test`.
```

Comment:
It would be better to say `BUILD_TEST` or `MOCK_TEST` is enabled, because it's possible to build by cmake directly, not using ./run.sh build.